src/Orleans/Configuration/MessagingConfiguration.cs

```diff
@@ -318,6 +333,29 @@ internal virtual void Load(XmlElement child)
                                                               "Invalid integer value for the MaxForwardCount attribute on the Messaging element");
                 }
             }
+
+            if (child.HasChildNodes)
+            {
+                var serializerNode = child.ChildNodes.Cast<XmlElement>().FirstOrDefault(n => n.Name == "SerializationProviders");
```

Comment:
Orleans Providers are very specific classes that inherit from IProvider.  Should we call these 'providers'?